pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "MIICE"
+version = "0.0.1"
+description = "Microscopy Image Illumination and Contrast Evaluation"
+readme = "README.md"
+license = "BSD-3-Clause"
+authors = ["Jenna Tomkinson <jenna.tomkinson@cuanschutz.edu>"]
+keywords = ["image-analysis", "cellpose", "microscopy", "illumination", "contrast"]
+packages = [{ include = "miice", from = "src" }]
+
+[tool.poetry.dependencies]
+python = ">=3.10,<4.0"
+matplotlib = "^3.10.0"
+cellpose = ">=4.0.6,<5.0"
+scikit-image = "^0.25.2"
+pandas = "^2.1.0"
+
+[tool.poetry.group.dev.dependencies]
+pre-commit = "^3.0.0"
+pytest = "^7.0.0"
+pytest-cov = "^4.0.0"
+black = "^24.0.0"
+isort = "^6.0.0"
+coverage = "^7.0.0"
+jupytext = "^1.15.0"
+jupyterlab = "^4.0.0"
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "-ra -q"
+testpaths = ["tests"]
```

Comment:
I didn't see the `tests` dir here. That said, I recommend creating tests for each of the existing functions to give a baseline level of coverage as you evolve the work.

Reply:
I can add that in another PR because I don't want to break anything yet lol. Let me add as an issue!